config/locales/it.yml

```diff
@@ -357,7 +358,7 @@ it:
       commentreplynotificationsettings: "Impostazioni di notifica della risposta a un commento"
       receiveemail: "Ricevi e-mail:"
       receivepushover: "Ricevi notifica Pushover:"
-      requirepushover: "Richiedi un iscrizione Pushover"
+      requirepushover: "Richiedi un'iscrizione Pushover qua sopra"
```

Comment:
qua sopra non mi suona benissimo. Di solito questi riferimenti posizionali sono confusionari. Come mai questa aggiunta?

Reply:
è così nell'originale. questo perché "sopra" c'è una sezione in cui è possibile collegare account esterni (GitHub, Twitter, Pushover). (Da noi questa sezione non c'è perché non abbiamo impostato le api key, però ci sarebbe)